juhpc

```diff
@@ -143,7 +143,9 @@ fi
 
 export JULIAUP_BINDIR="$JULIAUP_INSTALLDIR/bin" # juliaup and julia binaries
 
-if [ -d "/dev/shm" ]; then
+if [ ! -z "$NCAR_HOST"]; then
```

Comment:
in order to support your use case, I think it would be better to add a keyword argument `--tmp` and create internally in the variable `JULIAUP_TMP` (here:
https://github.com/JuliaParallel/JUHPC/blob/ff9247f28b2b4b199a6445331086ec1839ab1e3d/juhpc#L126-L139). Then, it can be assigned to `TMP` if it is set and else do as before.

Reply:
> Only other change was setting TMP directory to NCAR scratch (our scratch purges) though the solution might be applied differently for general future users.

By the way: why do you want to do that? Do you not have /tmp or /dev/shm on your system? Note that this is only used for a temporary installation of julia required to do the setup; the installation will be removed in the end.